packages/pds-ember/addon/components/pds/icon/index.hbs

```diff
@@ -1,3 +1,12 @@
-<div class="pds-icon" ...attributes data-test-root>
-  {{ svg-jar @type aria-hidden="true" }}
+<div
+  class="pds-icon"
+  ...attributes
+  data-test-root
+>
+  <div
+    class="pds-icon__svgWrapper"
+    data-test-wrapper
+  >
+    {{ svg-jar @type aria-hidden="true" }}
+  </div>
```

Comment:
An additional `<div>` was added around the SVG in order to properly inherit and apply `line-height` styling (which applies to _children_ of an element).